### DISCUSSION TITLE
How to get the inside config name in a config group?

### DISCUSSION THREAD

**Discussion by QingYunA** [facebookresearch/hydra#2803] [Q&A] [answered]
My config structure like:
conf
├── config
│   ├── my_conf1.yaml
│   └── my_conf2.yaml
└── config.yaml

in my `config.yaml`, I write:
```yaml
defaults:
  - config: my_conf1
  - _self_
```

and in `my_conf1.yaml`, i used code bellow to get the config name (expected to 'my_conf1.yaml')
```yaml
config_name: ${hydra:job.config_name}
```

however when i run my script file and print `config.config_name`
terminal output `config`

I am confused how to get the actually config name i used which should be "my_conf1"

**Comment by odelalleau** [comment, marked as answer]
Can’t try it right now but it’s something like `${hydra:runtime.choices.config}`

**Reply by QingYunA** [reply]
Thx, hope this feature can be added into newest version or add ${hydra:runtime.choices.config} into the hydra tutorials to help more people like me (newer to hydra).
